internal/generator/testdata/expected/normalDefinition

```diff
@@ -27,9 +27,9 @@ var directives = map[string][]uint{
     },
 }
 
-// Match is a matchFunc for parsing an NGINX config that contains the
+// MyMatchFn is a matchFunc for parsing an NGINX config that contains the
 // preceding directives.
```

Comment:
Nit: I think this testdata file is illustrating that the doc comment could be made more informative. As worded, the reader would need to look at the source to figure out what this function is doing.

Perhaps the generated functions are well named so it's not a big deal. Maybe there could be an optional flag that allows for some configuration of the doc string to say which nginx release or module this relates to?

Reply:
> Perhaps the generated functions are well named so it's not a big deal.

Yeah we would name it like `LuaDirectivesMatchFn` in `//go:generate ...`. This is just test data, not exact dynamic module so the name is kind of not that descriptive

> Maybe there could be an optional flag that allows for some configuration of the doc string to say which nginx release or module this relates to?

In `go generate` we will name the `MatchFunc` well. The tool will only be called directly when user wants to support some dynamic modules locally without MR. I thought if they want they can write comments directly? It's easier than pass comments to a command-line argument

Reply:
> I thought if they want they can write comments directly? It's easier than pass comments to a command-line argument
 
You mean edit-- and re-edit-- when the file is generated?

Adding a `--comment` option is one way to do it. Another way would be to allow the caller to provide a module name or some other string that gets dropped into the template. Forgive the pseudo syntax that follows.

```go
// {{.MatchFuncName}} is a matchFunc for parsing an NGINX config that contains
// directives from the {{.ModuleName}} module
```

You could add some logic to it or format it differently to add a bit more flexibility.

Reply:
Updated: add `match-func-comment` option